cmd/scanner/scanner.go

```diff
@@ -51,69 +47,53 @@ var (
 type Scanner struct {
 	inventory inventory.Inventory
 	stockers  []stocker.Stocker
-	apiURL    string
-	credsFile string
+	cfg       *config.ScannerConfig
 	logger    *zap.Logger
 }
 
 // NewScanner creates and returns a new Scanner instance
-func NewScanner(apiURL string, credsFile string, logger *zap.Logger) *Scanner {
+func NewScanner(cfg *config.ScannerConfig, logger *zap.Logger) *Scanner {
+	// Calculate Credentials file MD5 checksum for checking on runtime
+	hash := md5.Sum([]byte(cfg.CredentialsFile))
+	copy(hash[:], credsFileHash)
+
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+	client = http.Client{Transport: tr}
+
 	return &Scanner{
 		inventory: *inventory.NewInventory(),
 		stockers:  make([]stocker.Stocker, 0),
-		apiURL:    apiURL,
-		credsFile: credsFile,
+		cfg:       cfg,
 		logger:    logger,
 	}
 }
 
 func init() {
 	// Initialize logging configuration.
 	logger = ciqLogger.NewLogger()
-
-	// Load configuration from environment variables.
-	apiURL = os.Getenv("CIQ_API_URL")
-	credsFile = os.Getenv("CIQ_CREDS_FILE")
```

Comment:
I included the ENVVars in case we want to re-configure the scanner faster using env vars when running on OCP. Do you think still making sense? @vbelouso ?

Reply:
I think it's OK to leave it for now, it will be easy to remove if necessary